### ISSUE TITLE
options.files behaviour depends on location of package.json

### ISSUE THREAD

**Issue by lostfictions** [nwjs/nw-builder#145]
When I pass node-webkit-builder an array of globs in the `files` option (for example, something like `files: ['./src/**', './nwincludes/**']`), the zipped directory structure of the executable seems to depend on which of those folders contains `package.json`.

Given the example directories above, with `src/` containing `index.js`, and `nwincludes/` containing `index.html` and `logo.png`:

- If `src/` also contains `package.json`, the final directory structure looks like this:

    ```
index.js
package.json
nwincludes/index.html
nwincludes/logo.png
```

- If, on the other hand, `package.json` is in `nwincludes/`, the structure looks like this:

    ```
index.html
logo.png
package.json
src/index.js
```

In other words, the folder containing `package.json` becomes the root folder for the app, but other globbed directories are copied over with their relative paths intact. This is definitely unexpected behaviour -- I assumed all globbed directories would be copied in without relative paths.

**Comment by felixSchl** [comment]
Are you able to reproduce this in form of a failing [test](https://github.com/mllrsohn/node-webkit-builder/blob/master/test/utils.js#L78)?

**Comment by christophemarois** [comment]
+1, having the same problem
